packages/salesforcedx-visualforce-language-server/src/visualforceServer.ts

```diff
@@ -216,7 +216,9 @@ connection.onDidChangeConfiguration(change => {
   }
 });
 
-const pendingValidationRequests: { [uri: string]: NodeJS.Timer } = {};
+const pendingValidationRequests: {
+  [uri: string]: ReturnType<typeof setTimeout>;
```

Comment:
The timer type changes based on env.  Using the typeof makes it succeed regardless of env.